packages/gatsby-source-wordpress/src/steps/create-schema-customization/build-types.js

```diff
@@ -72,34 +70,20 @@ const interfaceType = typeBuilderApi => {
     extensions: { infer: false },
   }
 
-  // if this is a node interface type
-  if (nodeInterfaceTypes.includes(type.name)) {
-    // we add nodeType (post type) to all nodes as they're fetched
-    // so we can add them to node interfaces as well in order to filter
-    // by a couple different content types
-    typeDef.fields[`nodeType`] = `String`
```

Comment:
as mentioned above, I removed this field from the schema, it was a mistake that it was added and isn't needed.